internal/types/interface.go

```diff
@@ -161,12 +157,13 @@ func findSourcePackage(ident *ast.Ident, imports []*ast.ImportSpec) string {
 			continue
 		}
 
-		slash := strings.LastIndex(cleanPath, "/")
-		if ident.Name == cleanPath[slash+1:] {
+		matched, err := regexp.MatchString(`^(.*\/)?`+ident.Name+`(\/.[^\/]*)?$`, cleanPath)
```

Comment:
I agree with this

Reply:
Imo, if previous segment is matched, we have to check also the last part if it starts with 'v' and numbers. Otherwise it could be false positive match here

Reply:
I've replaced regexp with string comparing